.github/workflows/test-deploy-main.yml

```diff
@@ -257,6 +257,7 @@ jobs:
     environment: ci
     env:
       NODE_ENV: test
+
```

Comment:
With #1880 already merged, these workflow changes should be removed from this PR (same with `test-prs.yml`).

Reply:
Good catch! I forgot it. Done.